cmd/bnsd/app/app_performance_test.go

```diff
@@ -125,10 +129,25 @@ func newBnsd(t weavetest.Tester) abci.Application {
 	if err != nil {
 		t.Fatalf("cannot create a temporary directory: %s", err)
 	}
-	t.Logf("using home directory: %q", homeDir)
+	// t.Logf("using home directory: %q", homeDir)
 	bnsd, err := GenerateApp(homeDir, log.NewNopLogger(), false)
 	if err != nil {
 		t.Fatalf("cannot generate bnsd instance: %s", err)
 	}
 	return bnsd
 }
+
+func getNonce(db weave.ReadOnlyKVStore, addr weave.Address) (int64, error) {
```

Comment:
:+1:  

How about calling it `Nonce`? This would make it public and remove `get` as [it does not provide any additional information](https://golang.org/doc/effective_go.html#Getters).

I did not implement getting a nounce because I know that Alex have implemented the very same functionality in the bnds [test client package](https://github.com/iov-one/weave/blob/master/cmd/bnsd/client/client.go#L69). As we progress with those tests more and more client will be reimplemented. That made me wonder how to not repeat too much testing code.

I like nonce getter as a function. I only wonder if implementing the same functionality as in acceptance tests is good enough for now.

Reply:
Yeah, this is a simpler version of his client package one, but demoing use of the KVStore wrapping.

I will extend it to use the full power of Nonce from client